server/sonar-server/src/main/java/org/sonar/server/usergroups/ws/UpdateAction.java

```diff
@@ -122,4 +134,16 @@ private void updateDefaultGroupIfNeeded(DbSession dbSession, String oldName, Str
       persistentSettings.saveProperty(dbSession, CORE_DEFAULT_GROUP, newName);
     }
   }
+
+  private void writeResponse(DbSession dbSession, Request request, Response response, OrganizationDto organization, GroupDto group) {
+    UserMembershipQuery query = UserMembershipQuery.builder()
+      .groupId(group.getId())
+      .membership(GroupMembershipQuery.IN)
+      .build();
+    int membersCount = dbClient.groupMembershipDao().countMembers(dbSession, query);
+
+    WsUserGroups.UpdateResponse.Builder respBuilder = WsUserGroups.UpdateResponse.newBuilder();
+    respBuilder.setGroup(support.toProtobuf(organization, group, membersCount));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Change this instance-reference to a static reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2209)